solidity/test/LPRewardsTest.js

```diff
@@ -0,0 +1,220 @@
+const {contract, web3, accounts} = require("@openzeppelin/test-environment")
+const {createSnapshot, restoreSnapshot} = require("./helpers/snapshot")
+const {time} = require("@openzeppelin/test-helpers")
+
+const LPRewards = contract.fromArtifact("LPRewards")
+const KeepToken = contract.fromArtifact("KeepToken")
+const WrappedToken = contract.fromArtifact("TestToken")
+
+const BN = web3.utils.BN
+const chai = require("chai")
+chai.use(require("bn-chai")(BN))
+const expect = chai.expect
+
+describe("LPRewards", () => {
+  const tokenDecimalMultiplier = web3.utils.toBN(10).pow(web3.utils.toBN(18))
+
+  let keepToken
+  let lpRewards
+  let wrappedToken
+  let owner
+  let wallet1
+  let wallet2
+
+  before(async () => {
+    owner = accounts[0]
+    wallet1 = accounts[1]
+    wallet2 = accounts[2]
+    keepToken = await KeepToken.new()
+    // This is a "Pair" Uniswap Token which is created here:
+    // https://github.com/Uniswap/uniswap-v2-core/blob/master/contracts/UniswapV2Factory.sol#L23
+    //
+    // There are 3 addresses for the following pairs:
+    // - KEEP/ETH (https://info.uniswap.org/pair/0xe6f19dab7d43317344282f803f8e8d240708174a)
+    // - TBTC/ETH (https://info.uniswap.org/pair/0x854056fd40c1b52037166285b2e54fee774d33f6)
+    // - KEEP/TBTC (https://info.uniswap.org/pair/0x38c8ffee49f286f25d25bad919ff7552e5daf081)
+    wrappedToken = await WrappedToken.new()
+    lpRewards = await LPRewards.new(keepToken.address, wrappedToken.address)
+  })
+
+  beforeEach(async () => {
+    await createSnapshot()
+  })
+
+  afterEach(async () => {
+    await restoreSnapshot()
+  })
+
+  describe("tokens allocation", () => {
+    it("should successfully allocate KEEP tokens", async () => {
+      let keepBalance = await keepToken.balanceOf(lpRewards.address)
+      expect(keepBalance).to.eq.BN(0)
+
+      const rewards = web3.utils.toBN(1000042).mul(tokenDecimalMultiplier)
+      await fundKEEPReward(lpRewards.address, rewards)
```

Comment:
Let's add a line break to separate parameter configuiration from the actual test function call.
```suggestion
      const rewards = web3.utils.toBN(1000042).mul(tokenDecimalMultiplier)
      
      await fundKEEPReward(lpRewards.address, rewards)
```

Reply:
done